docs/demos/authentication/README.md

```diff
@@ -0,0 +1,113 @@
+
+## Configure Azure Active Directory
+
+1. Sign in to the [Azure Portal](https://portal.azure.com)
+1. Select the `Azure Active Directory` option from the side menu
+
+   ![select active directory](images/select-active-directory.png)
+1. Select the `App registrations` option from the Azure Active Directory side menu.
+   
+   ![select app registrations](images/select-app-registrations.png)
+1. Select the `New registration` option from the Azure Active Directory top menu.
+   
+   ![select new registration](images/select-new-registration.png)
+1. On the `Register an application` page, provide a name for the application and select `Register`.
+   
+   ![provide application name and select register](images/provide-application-name.png)
+1. On the app registration overview page, take note of the `Application (client) ID` and `Directory (tenant) ID` as we will need this information for a later step.
+1. Select the `Manage > App Roles` side menu option from the app registration management page.
+   
+   ![select manage > app roles](images/select-app-roles.png)
+1. Select the `Create app role` option from the App roles top menu.
+   
+   ![create app role](images/select-create-app-role.png)
+1. Configure the app role with the following information and select `Apply`.
+   
+   ![create app role](images/create-app-role.png)
+
+   | Configuration                        | Value                                |
+   | ------------------------------------ | ------------------------------------ |
+   | Display Name                         | `API Access`                         |
+   | Allowed member types                 | `Both (Users/Groups + Applications)` |
+   | Value                                | `API.Access`                         |
+   | Description                          | `Allows a user to edit in the api.`  |
+   | Do you want to enable this app role? | `Enabled`                            |
+
+## Enable Authentication in the Podcast.API
+
+1. In the `Podcast.API` project, open the `appsettings.json` (or environment specific file such as `appsettings.Development.json`).
+1. Add a new top-level property of `AzureAd` with the following configurations. The `{DOMAIN}`, `{AZURE_AD_TENANT_ID}`, and `{AZURE_AD_CLIENT_ID}` will need to be replaced with values captured earlier when configuring your app registration.
+
+   ```json
+   {
+     "Logging": {
+       "LogLevel": {
+         "Default": "Information",
+         "Microsoft": "Warning",
+         "Microsoft.Hosting.Lifetime": "Information"
+       }
+     },
+     "ConnectionStrings": {
+       "PodcastDb": "Server=localhost, 5433;Database=Podcast;User Id=sa;Password=Pass@word;Encrypt=False",
+       "FeedQueue": "UseDevelopmentStorage=true"
+     },
+     "AzureAd": {
+       "Instance": "https://login.microsoftonline.com/",
+       "Domain": "{DOMAIN}",
+       "TenantId": "{AZURE_AD_TENANT_ID}",
+       "ClientId": "{AZURE_AD_CLIENT_ID}"
+     },
+     "Authentication": {
```

Comment:
We may want to remove this config section and show that it gets populated after the user runs `dotnet user-jwts` below.

Reply:
Correct. Just the authentication section needs to be removed.

Reply:
@michaelstonis any update on this?